### PR TITLE
Terms of use strings

### PR DESCRIPTION
**Pull request by alexzatsepin**

### CODE REVIEW

**Review by keksikex — APPROVED** [review]
LGTM

File: data/strings/strings.txt [hunk 1]
```diff
@@ -48743,3 +48742,40 @@
 
   [sberbank_map_downloader_cta_button]
     en = Узнать больше
+
+  [upd_licagr_popup]
+    en = We are glad to inform you that MAPS.ME is now owned and operated by MAPS.ME (Cyprus) Ltd. We have updated our <a href=\"%s\">Terms of Use and End User License Agreement</a> and <a href=\"%s\">Privacy Policy</a> due to the assignment of all rights and obligations in relation to MAPS.ME to MAPS.ME (CYPRUS) LTD. By continuing using our services, you agree to these <a href=\"%s\">Terms of use</a> and <a href=\"%s\">Privacy Policy</a>
```
Comment (Polas) [inline data/strings/strings.txt:48747]: Слеши перед кавычками не нужны, их проставит генератор

Comment (alexzatsepin) [reply data/strings/strings.txt:48747]: Fixed

**Comment by alexzatsepin** [comment]
@milchakov ids are renamed, fyi.

**Comment by alexzatsepin** [comment]
@Polas ptal

**Comment by milchakov** [comment]
@alexzatsepin Thank you very much but you forget about underscores))

**Comment by alexzatsepin** [comment]
> @alexzatsepin Thank you very much but you forget about underscores))

@milchakov Good catch! I'll fix it.

**Comment by alexzatsepin** [comment]
Added underscore. @milchakov @Polas ptal

**Comment by milchakov** [comment]
lgtm

**Review by Polas — APPROVED** [review]

**Pull request merged**